docs/qaoa_example.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Optional: Could mention same as the other MPS doc (and link to it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asciineuron/bgls/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>